README.md

```diff
@@ -53,3 +53,7 @@ If you build your module with this command:
 You could generate `compile_commands.json` with:
 
     $ python .vscode/generate_compdb.py -O $KDIR $PWD
+
+Ensure that your module has been built and contains some `*.o.cmd` files before generating `compile_commands.json`.
```

Comment:
This is also true for kernel sources too (4th line of README.md). Is it really necessary to repeat this for modules?

Reply:
What messed me up was the phrase "ensure the kernel is built". I interpreted that as "for modules, only the kernel needs to be built, not the module". Now that I know a little more about how this script works and the contents of a .cmd file, I know this wasn't a reasonable interpretation, but I thought this note might be helpful for fellow kernel newbies. Feel free to take it or leave it.